smartlogic/healthcheck.go

```diff
@@ -90,36 +101,50 @@ func (h *SmartlogicConcordanceTransformerHandler) concordanceRwNeo4jHealthCheck(
 	}
 }
 
-func (h *SmartlogicConcordanceTransformerHandler) checkConcordanceRwConnectivity() (string, error) {
+func (h *ConcordanceTransformerHandler) checkConcordanceRwConnectivity() (string, error) {
 	urlToCheck := h.transformer.writerAddress + "__gtg"
 	request, err := http.NewRequest("GET", urlToCheck, nil)
 	if err != nil {
 		clientError := fmt.Sprintf("Error creating request to writer %s : %v", urlToCheck, err)
-		log.WithError(err).Error(clientError)
+		h.log.WithError(err).Error(clientError)
 		return clientError, errors.New("Unable to verify availibility of concordances-rw-neo4j")
 	}
 	resp, err := h.transformer.httpClient.Do(request)
 	if err != nil {
 		clientError := fmt.Sprintf("Error calling writer at %s : %v", urlToCheck, err)
-		log.WithError(err).Error(clientError)
+		h.log.WithError(err).Error(clientError)
 		return clientError, errors.New("Unable to verify availibility of concordances-rw-neo4j")
 	}
-	defer resp.Body.Close()
+	defer func(Body io.ReadCloser) {
+		err := Body.Close()
+		if err != nil {
+			h.log.WithError(err).Info("Could not close body")
+		}
+	}(resp.Body)
 	if resp != nil && resp.StatusCode != http.StatusOK {
```

Comment:
It can be misleading to use the `Body` field of the response and do this check below so let's improve this by:

```suggestion
	if resp.StatusCode != http.StatusOK {
```